symbolic-symcache/Cargo.toml

```diff
@@ -35,7 +35,6 @@ watto = { workspace = true }
 insta = { workspace = true }
 criterion = { workspace = true }
 symbolic-testutils = { workspace = true }
-similar-asserts = { workspace = true }
```

Comment:
you can probably remove that from the main `Cargo.toml` now?

Reply:
`similar-asserts` exist in `symbolic-cfi`